modules/op-chains-signing-secrets-ecdsa.adoc

```diff
@@ -0,0 +1,53 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="chains-signing-secrets-ecdsa_{context}"]
+= Generating the x509 key pair
```

Comment:
I think you should add an explanation why this key pair is needed

Reply:
Reading further, maybe just change the header to "Generating the key pair to use the x509 signing scheme"? And/or add at the start "To use the x509 signing scheme for {tekton-chains} secrets, you must generate the x509 key pair".

Of course this would need SME review as my guess can be wrong. But one way or the other the user needs context here in my view.

Reply:
Hello Misha, yes, I am actually raising this question about the heading [here](https://github.com/openshift/openshift-docs/pull/84253#discussion_r1849897787) so we are definitely on the same page :+1:

Reply:
cc @PuneetPunamiya

Reply:
Great, that sound good, thank you! :slightly_smiling_face: